### PR TITLE
Update CI to use Python 3.5 and switch to Buildkite

### PR DESCRIPTION
**Pull request by reivilibre**
Prior to this, it used Python 2.7.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = packaging, pep8, py27-ldap{0,1,2}
+envlist = packaging, pep8, py35-ldap{0,1,2}
```
Comment (clokep) [inline tox.ini:2]: It looks like setup.py says only `ldap>=2.8` -- is it still worth testing these old versions?

Also, would probably be good to run `py36`, `py37` (and maybe `py38`)?

**Comment by reivilibre** [comment]
I'm now using the tox-travis plugin which seems to work quite well, is that fine?

**Comment by clokep** [comment]
Although it was brought up of whether we should be converting this to buildkite instead?

**Comment by reivilibre** [comment]
Switched to Buildkite. Fail above is because the pipeline isn't enabled yet.

**Pull request merged**